packages/ui-components/src/components/menu.ts

```diff
@@ -14,7 +13,7 @@ import { Menu, Widget } from '@lumino/widgets';
  * application menus that may be extended by plugins as well,
  * such as "Edit" and "View"
  */
-export interface IJupyterLabMenu extends IDisposable {
+export interface IRankedMenu extends IDisposable {
```

Comment:
Could it be that moving that to another package might break for those importing `IJupyterLabMenu`? for example with:

```typescript
import { IJupyterLabMenu } from '@jupyterlab/mainmenu';
```

Reply:
It is still exported with the old name in `@jupyterlab/mainmenu` (index.ts#L22-L25):

```ts
export {
  IRankedMenu as IJupyterLabMenu,
  RankedMenu as JupyterLabMenu
} from '@jupyterlab/ui-components';
```

So downstream packages should be compatible.

Reply:
Oh nice, sorry I missed that.

Thanks!

Reply:
No problem this is a small set of lines in a not small PR :wink: